vms/platformvm/service.go

```diff
@@ -527,6 +528,47 @@ type GetSubnetsResponse struct {
 	Subnets []APISubnet `json:"subnets"`
 }
 
+func (s *Service) GetSubnet(_ *http.Request, args *api.GetSubnetArgs, response *api.GetTxReply) error {
+	s.vm.ctx.Log.Debug("API called",
+		zap.String("service", "platform"),
+		zap.String("method", "getSubnet"),
+	)
+
+	subnetID := args.SubnetID
+
+	var tx *txs.Tx
+	var err error
+	tx, err = s.vm.state.GetSubnetTransformation(subnetID)
+	if err != nil && err != database.ErrNotFound {
+		return fmt.Errorf("couldn't get transformed subnet: %w", err)
+	}
+	if err != nil && err == database.ErrNotFound {
```

Comment:
We don't need to recheck `err !=nil` here before `err == database.ErrNotFound`

Reply:
done